js/background.js

```diff
@@ -398,17 +398,19 @@
     await storage.put('version', currentVersion);
 
     if (newVersion) {
+      await window.Signal.Data.cleanupOrphanedAttachments();
+
+      window.log.info(
+        `New version detected: ${currentVersion}; previous: ${lastVersion}`
+      );
+
       if (
         lastVersion &&
         window.isBeforeVersion(lastVersion, 'v1.15.0-beta.5')
```

Comment:
Looks like this line logic was there for when Signal moved to SQLCipher. I don't think we need it in Session as our product started off using SQLCipher anyway. Also `window.isBeforeVersion` will always return `true` for our current Session version. Thoughts on just removing this? @neuroscr @konstantinullrich @msgmaxim

Reply:
Makes sense to me

Reply:
but we've always triggered this condition in Messenger. To me, Log clean up seems important in a secure messenger, so I'd rather have it than not.

Reply:
But I think @Mikunj is referring to the if statement, which is redundant.

Reply:
removing the if makes a lot of sense

Reply:
I could also avoid future bugs if we pass version 1.15.0

Reply:
Yep i was talking about the `if` condition. Do we want to remove the logs on each start up? Signal has an implementation where users can view their logs and submit them if they need assistance. Do we want to be able to use that somehow in the future?